test/extended/testdata/testjob-plugin.xml

```diff
@@ -53,16 +69,62 @@
       <authToken></authToken>
     </com.openshift.jenkins.plugins.pipeline.OpenShiftServiceVerifier>
 
+    <!-- 1 to 1 mapping -->
     <com.openshift.jenkins.plugins.pipeline.OpenShiftImageTagger>
       <apiURL>https://openshift.default.svc.cluster.local</apiURL>
       <namespace>PROJECT_NAME</namespace>
+      <testStream>origin-nodejs-sample</testStream>
       <testTag>latest</testTag>
+      <prodStream>origin-nodejs-sample</prodStream>
       <prodTag>prod</prodTag>
+      <authToken></authToken>
+    </com.openshift.jenkins.plugins.pipeline.OpenShiftImageTagger>
+
+    <!-- 1 to N mapping -->
+    <com.openshift.jenkins.plugins.pipeline.OpenShiftImageTagger>
+      <apiURL>https://openshift.default.svc.cluster.local</apiURL>
+      <namespace>PROJECT_NAME</namespace>
       <testStream>origin-nodejs-sample</testStream>
+      <testTag>latest</testTag>
       <prodStream>origin-nodejs-sample</prodStream>
+      <prodTag>prod2,prod3,prod4</prodTag>
```

Comment:
@gabemontero do we support both "source/destination" naming conventions as well as this "test/prod" convention?  and if so, we should be stripping out usage of the test/prod syntax, not adding more.